### PR TITLE
Sprinkles: Clean up public API and document on site

### PR DESCRIPTION
**Pull request by michaeltaranto**
See changeset

### CODE REVIEW

File: .changeset/nervous-pumpkins-matter.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+'@vanilla-extract/sprinkles': minor
+---
+
+Clean up public API, deprecating old API names. Also adding sprinkles to the docs site and using `sprinkles` in favour of `atoms` for the canoncial examples.
+
+API changes include:
+- Rename `createAtomicStyles` to `defineProperties`, `createAtomicStyles` is now deprecated
+- Rename `createAtomsFn` to `createSprinkles`, `createAtomsFn` is now deprecated
+- Rename `AtomicStyles` type to `SprinklesStyles`, `AtomicStyles` is now deprecated
```
Comment (markdalgleish) [inline .changeset/nervous-pumpkins-matter.md:10]: Should this type be `SprinklesProperties`?

File: README.md [hunk 2]
```diff
@@ -1016,26 +1016,26 @@ document.write(`
 
 Your recipe configuration can also make use of existing variables, classes and styles.
 
-For example, you can use your `atoms` function from [Sprinkles.](https://github.com/seek-oss/vanilla-extract/tree/master/packages/sprinkles)
+For example, you can pass in the result of your [sprinkles](https://vanilla-extract.style/documentation/sprinkles-api) function directly.
```
Comment (markdalgleish) [inline README.md:1019]:
```suggestion
For example, you can pass in the result of your [`sprinkles`](https://vanilla-extract.style/documentation/sprinkles-api) function directly.
```

File: packages/sprinkles/src/createSprinkles.ts [hunk 3]
```diff
@@ -183,23 +180,21 @@ export const createAtomsFn =
               );
             };
 
-            if (!atomicProperty) {
-              throw new SprinklesError(
-                `"${prop}" is not a valid atom property`,
-              );
+            if (!sprinkle) {
+              throw new SprinklesError(`"${prop}" is not a valid sprinkle`);
```
Comment (markdalgleish) [inline packages/sprinkles/src/createSprinkles.ts:184]: Heh, this error is cute.

File: packages/sprinkles/src/createUtils.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { addRecipe } from '@vanilla-extract/css/recipe';
+import { addFunctionSerializer } from '@vanilla-extract/css/functionSerializer';
```
Comment (markdalgleish) [inline packages/sprinkles/src/createUtils.ts:1]: Not sure we should change this as it'll break for consumers running older versions of the integration package for no benefit.

File: packages/sprinkles/src/index.ts [hunk 5]
```diff
@@ -4,14 +4,14 @@ import {
   CSSProperties,
   StyleRule,
 } from '@vanilla-extract/css';
-import { addRecipe } from '@vanilla-extract/css/recipe';
```
Comment (markdalgleish) [inline packages/sprinkles/src/index.ts:7]: Same as above.

File: site/docs/recipes-api.md [hunk 6]
```diff
@@ -82,27 +82,27 @@ document.write(`
 
 Your recipe configuration can also make use of existing variables, classes and styles.
 
-For example, you can use your `atoms` function from [Sprinkles.](https://github.com/seek-oss/vanilla-extract/tree/master/packages/sprinkles)
+For example, you can pass in the result of your [sprinkles](/documentation/sprinkles-api) function directly.
```
Comment (markdalgleish) [inline site/docs/recipes-api.md:85]:
```suggestion
For example, you can pass in the result of your [`sprinkles`](/documentation/sprinkles-api) function directly.
```

File: .changeset/nervous-pumpkins-matter.md (hunk 1 above)
Comment (michaeltaranto) [reply .changeset/nervous-pumpkins-matter.md:10]: Yeah actually that sounds obvious now that you say it 👍

File: site/src/App.css.ts [hunk 7]
```diff
@@ -10,6 +10,7 @@ globalStyle('html, body', {
 globalStyle(`.${darkMode}`, {
   background: vars.palette.black,
   color: vars.palette.gray50,
+  colorScheme: 'dark',
```
Comment (markdalgleish) [inline site/src/App.css.ts:13]: Nice one 👏

File: packages/sprinkles/src/createUtils.ts (hunk 4 above)
Comment (mattcompiles) [reply packages/sprinkles/src/createUtils.ts:1]: Yeah this needs to stay as `addRecipe` for the time being

File: site/docs/recipes-api.md (hunk 6 above)
Comment (michaeltaranto) [reply site/docs/recipes-api.md:85]: I'll push this separately, the site doesn't support inline code as links very well.

**Review by markdalgleish — APPROVED** [review]
Awesome work 👏

**Pull request merged**